crates/driver/src/run.rs

```diff
@@ -150,7 +150,12 @@ fn solvers(config: &config::Config, eth: &Ethereum) -> Vec<Solver> {
     config
         .solvers
         .iter()
-        .map(|config| Solver::new(config.clone(), eth.clone()))
+        .map(|config| {
+            match Solver::new(config.clone(), eth.clone()) {
+                Ok(solver) => solver,
+                Err(err) => panic!("Could Not initialize solver: {:?}", err)
+            }}
```

Comment:
```suggestion
            Solver::new(config.clone(), eth.clone()).unwrap()
```

Reply:
What's the purpose of making it return a result if you're just gonna unwrap it anyways without checking?

Reply:
[Unwrap](https://doc.rust-lang.org/src/core/result.rs.html#1067-1069) does exactly what you wrote here (check the value and panic if it's not Ok)

Also the important bit is where we `unwrap` (in run.rs which is obviously called on startup or some obscure constructor which may be called at runtime causing an up and running system to panic, which we want to avoid)

Reply:
I knew that it panics if there's an error but I guess I just assumed you would want a more comprehensive error thrown than just unwrap which may be harder to debug. But if it's fine with you it's fine with me.

Reply:
Can you argue how your error message is more comprehensive than `unwrap`'s [implemantation](https://doc.rust-lang.org/src/core/result.rs.html#1649)?

Reply:
I guess I had thought that an error message that says `solver failed to initialize` would be more useful than a generic `failed to unwrap result` but you're the expert here so i'll do whatever you want.

Reply:
Sorry for the delays, been at ethdenver so haven't really had a chance to work on this. Will finish it up this week

Reply:
I've made the requested change.